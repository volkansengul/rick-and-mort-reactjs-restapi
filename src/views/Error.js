import React from 'react';
import {Link} from 'react-router-dom';

export default class Error extends React.Component {
    render() {
        return (
            <div className={'error'}>
                <h1>Ups !</h1>
                <p>Something went wrong! Please try again later.</p>
                <Link to={'/'} className={'tryit-btn'}>Try it now :)</Link>
            </div>
        )
    }
}