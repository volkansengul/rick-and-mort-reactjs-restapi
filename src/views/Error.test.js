import React from 'react';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';

import Error from './Error';

it('Error page renders without crashing', () => {
    const episodeList = renderer.create(
        <Router>
            <Error/>
        </Router>
    ).toJSON();
    expect(episodeList).toMatchSnapshot();
});
