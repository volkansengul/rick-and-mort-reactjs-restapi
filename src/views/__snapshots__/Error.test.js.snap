// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page renders without crashing 1`] = `
<div
  className="error"
>
  <h1>
    Ups !
  </h1>
  <p>
    Something went wrong! Please try again later.
  </p>
  <a
    className="tryit-btn"
    href="/"
    onClick={[Function]}
  >
    Try it now :)
  </a>
</div>
`;
